.eslintrc

```diff
@@ -17,16 +17,18 @@ rules:
   strict: 2
   # We actually have a transform to support this and we fix this for bundled
   # releases but not for the npm package, so enforce it strictly
-  no-comma-dangle: 2
+  comma-dangle: [2, never]
```

Comment:
In a followup, let's stop enforcing this (and maybe do the opposite). This does get transformed correctly now.